.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+
+node_js:
+  - '10'
+
+install: yarn --frozen-lockfile
+
+cache:
+  yarn: true
+  directories:
+    - 'node_modules'
+    - '.cache'
+
+script:
+  - yarn run test
```

Comment:
`yarn test` would also work here

Reply:
Yes It would work but I would prefer the official version and not an alias. https://yarnpkg.com/en/docs/cli/run